nextcord/scheduled_events.py

```diff
@@ -24,28 +24,26 @@
 
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Tuple, Union
 
 from .abc import Snowflake
 from .asset import Asset
 from .enums import ScheduledEventPrivacyLevel
 from .iterators import ScheduledEventUserIterator
 from .mixins import Hashable
-from .utils import MISSING, _bytes_to_base64_data, parse_time
+from .utils import MISSING, _obj_to_base64_data, parse_time
 
-__all__: Tuple[str, ...] = (
-    "EntityMetadata",
-    "ScheduledEventUser",
-    "ScheduledEvent",
-)
+__all__: Tuple[str, ...] = ("EntityMetadata", "ScheduledEventUser", "ScheduledEvent")
```

Comment:
Removing this comma doesn't seem necessary
```suggestion
__all__: Tuple[str, ...] = (
    "EntityMetadata",
    "ScheduledEventUser",
    "ScheduledEvent",
)
```